README.md

```diff
@@ -1,6 +1,12 @@
-# beef board
+# beef board<a name="intro"></a>
 [![Continuous Integration](https://github.com/HWXLR8/beef-board/actions/workflows/ci.yml/badge.svg)](https://github.com/HWXLR8/beef-board/actions/workflows/ci.yml)
 
+1. [Wiring configuration](#wiring-configuration)
+2. [Settings](#settings)
+3. [SpiceTools companion scripts](#spicetools)
+4. [Bill of Materials](#bom)
+5. [Credits](#credits)
+
```

Comment:
could we please move this below the introduction? Similar to wikipedia style:

Brief intro
Table of contents
...the rest of the article.

I would just like the intro front and center to quickly answer the question of "what is this thing".